frontend/src/components/nodes/NodeNav.tsx

```diff
@@ -26,8 +26,8 @@ export default class extends React.Component<Props> {
                 >
                   <Col className="node-selector align-self-center">
                     {context.nodeInfo.validatorAmount
-                      ? `${context.nodeInfo.validatorAmount}  Validating`
-                      : `- Validating`}
+                      ? `${context.nodeInfo.validatorAmount}  Validating & New Upcoming`
+                      : `- Validating & New Upcoming`}
```

Comment:
Here is how you can avoid such bugs from code duplication:

```js
{`${context.nodeInfo.validatorAmount ?? '-'} Validating & New Upcoming`}
```

Read more about ["#3: Nullish Coalescing"](https://www.freecodecamp.org/news/javascript-new-features-es2020/)

Reply:
thanks!